app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreen.kt

```diff
@@ -441,6 +455,125 @@ private fun ImportLoginsStepThreeContent(
+                        navigationIconContentDescription = stringResource(R.string.close),
+                    ),
+                    scrollBehavior = scrollBehavior,
+                )
+            },
+            modifier = Modifier
+                .nestedScroll(scrollBehavior.nestedScrollConnection)
+                .fillMaxSize(),
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(paddingValues = it)
+                    .verticalScroll(rememberScrollState()),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                Spacer(Modifier.height(24.dp))
+                Image(
+                    painter = rememberVectorPainter(R.drawable.img_secure_devices),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .standardHorizontalMargin()
+                        .size(124.dp),
+                )
+                Spacer(Modifier.height(24.dp))
+                Text(
+                    text = stringResource(R.string.import_successful),
+                    style = BitwardenTheme.typography.titleMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.standardHorizontalMargin(),
+                )
+                Spacer(Modifier.height(12.dp))
+                Text(
+                    text = stringResource(
+                        R.string.manage_your_logins_from_anywhere_with_bitwarden_tools,
+                    ),
+                    style = BitwardenTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.standardHorizontalMargin(),
+                )
+                Spacer(Modifier.height(24.dp))
+                @Suppress("MaxLineLength")
+                BitwardenContentCard(
+                    contentItems = persistentListOf(
+                        ContentBlockData(
+                            headerText = stringResource(R.string.download_the_browser_extension),
+                            subtitleText = stringResource(R.string.go_to_bitwarden_com_download_to_integrate_bitwarden_into_browser),
+                            iconResource = R.drawable.ic_puzzle,
+                        ),
+                        ContentBlockData(
+                            headerText = stringResource(R.string.use_the_web_app),
+                            subtitleText = stringResource(
+                                R.string.log_in_at_bitwarden_com_to_easily_manage_your_account,
+                            ),
+                            iconResource = R.drawable.ic_desktop,
+                        ),
+                        ContentBlockData(
+                            headerText = stringResource(R.string.autofill_passwords),
+                            subtitleText = stringResource(R.string.set_up_autofill_on_all_your_devices),
```

Comment:
Can we do this to not break the line length here:
```kotlin
subtitleText = stringResource(
      R.string.set_up_autofill_on_all_your_devices,
 ),
```

Reply:
this was the straw, and it still exceeds it like this,
```
subtitleText = stringResource(
                                R.string.go_to_bitwarden_com_download_to_integrate_bitwarden_into_browser,
                            ),
```
could chain it... but personally the suppression is less awful than 
```
subtitleText = stringResource(
                                R
                                    .string
                                    .go_to_bitwarden_com_download_to_integrate_bitwarden_into_browser,
                            ),